source/_integrations/frigidaire.markdown

```diff
@@ -0,0 +1,29 @@
+---
+title: Frigidaire
+description: Instructions on how to integrate Frigidaire smart AC within Home Assistant.
+ha_category:
+  - Presence Climate
+ha_release: 2021.06
+ha_iot_class: Cloud Polling
+ha_quality_scale: silver
```

Comment:
This is speculative, let me know if this is not the right thing to do and I can remove this line